features/guidance/dedicated-turn-roads.feature

```diff
@@ -71,19 +71,19 @@ Feature: Slipways and Dedicated Turn Lanes
     Scenario: Turn Instead of Ramp
         Given the node map
             """
-                    e
-            a b     c d
-                  h
-
-
-
-
-
-
-
+                    e---.
+            a-b-----c-d |
+               `--h     |
+                   \    /
+                    |  /
+                    | /
+                    |/
+                    |
+                    |
+                    |
                     f
```

Comment:
This should not have a detour for e--f. They should go straight under `c`. Sorry for not being clear here
A better description of the case would be
```
                    e
                   \_/
a-b - - - - - - - - c - d
                   /-\
                    |
```

Note that changing the location of `e` does not influence the case (a lot) since only one segment gets a bit longer. turn angles remain the same. It is not a change that is save in general, but here it is.
Also below.